CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -103,8 +103,8 @@
         <note>How many episodes a person has been appeared in for a TV show, or how many episodes are in a TV season</note>
       </trans-unit>
       <trans-unit id="%lld free search%@ remaining" xml:space="preserve">
-        <source>%1$lld free search%2$@ remaining</source>
-        <note>Shown on the main/bottom bar when the user has fewer than 10 of their free searches left</note>
+        <source>%1$lld free search%2$@ remaining</source><target state="needs-review-translation">%1$lld verbleibende Suchanfrage(n)%2$@</target>
+        <note>Shown on the main/bottom bar when the user has fewer than 10 of their free searches left</note><note from="reviewer">Should this not have a one/many variant? Worked around it with the n in brackets, but this is not pretty</note>
```

Comment:
So this was also raised as an Issue by someone else here, I think in the Ukrainian fork.

It's "1 verbleibende Suchanfrage" and "2 verbleibende Suchanfragen"